test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -11,8 +11,28 @@
 
 
 def _read_file(path):
+    with open(path, 'r') as fp:
+        return fp.read().replace('\r', '').replace('\\r', '')
+
+
+class LineIndifferent(object):
+    def __init__(self, expected):
+        self.expected = expected.replace('\r', '')
+
+    def __eq__(self, other):
+        return self.expected == other.replace('\r', '')
+
+    def __repr__(self):
+        return 'LineIndifferent("self.expected")'
```

Comment:
do you want the literal `"self.expected"` here, or should that be string formatted?

Reply:
that's definitely a mistake on my part!